src/CommaSeparatedJsonParser.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace MPScholten\RequestParser;
+
+class CommaSeparatedJsonParser extends AbstractValueParser
+{
+    protected function parse($value)
+    {
+        if (!empty($value) && $value[0] !== '[' && $value[strlen($value) - 1] !== ']') {
```

Comment:
Ah I see your point. I was actually delivering it like that by design 😃 
So perhaps we just decorate it with square brackets anyway...